docs/storage.md

```diff
@@ -418,6 +418,23 @@ config:
 
 Use --objstore.config-file to reference to this configuration file.
 
+#### Baidu BOS
+
+In order to use Baidu BOS object storage, you should apply a Baidu Account to create an object storage bucket at first. Note that detailed from Baidu Cloud Documents:[https://cloud.baidu.com/doc/BOS/index.html](https://cloud.baidu.com/doc/BOS/index.html) for more detail.
+
+To use Baidu BOS object storage, please specify following yaml configuration file in `objstore.config*` flag.
+
+```yaml mdox-exec="go run scripts/cfggen/main.go --name=bos.Config"
+type: BOS
+config:
+  bucket: ""
+  endpoint: ""
+  access_key: ""
+  secret_key: ""
+```
+
+Use --objstore.config-file to reference to this configuration file.
+
```

Comment:
```suggestion
```

Not sure what this sentenced tried to say. I think it's fine whatever we have above